tutorials/basic-topics/sorting/bucket-sort.md

```diff
@@ -1,14 +1,316 @@
 ---
+
 title: 'Bucket Sort'
-description: ''
+description: 'Bucket sort is a sorting algorithm that organizes elements into buckets based on their values and sorts each bucket separately for efficient sorting.'
 hide_table_of_contents: true
 draft: true
 keywords:
-  - leetcode
-  - tutorial
-  - sorting
-  - algorithm
-  - bucket sort
+
+- leetcode
+- tutorial
+- sorting
+- algorithm
+- bucket sort
+
 ---
 
-<TutorialAuthors names="@TBC"/>
+<TutorialAuthors names="@Sanchita1304"/>
+<Contributors names="@wingkwong"/>
```

Comment:
i didn't contribute. please remove this line.

Reply:
okk